apps/backend/src/modules/users/user.service.ts

```diff
@@ -23,15 +30,22 @@ class UserService implements Service {
 	): Promise<UserSignUpResponseDto> {
 		const { hash, salt } = await this.encrypt.encrypt(payload.password);
 
-		const item = await this.userRepository.create(
+		const user = await this.userRepository.create(
 			UserEntity.initializeNew({
 				email: payload.email,
+				firstName: payload.firstName,
+				lastName: payload.lastName,
 				passwordHash: hash,
 				passwordSalt: salt,
 			}),
 		);
 
-		return item.toObject();
+		const userObject = user.toObject();
+
+		return {
+			token: this.token.create({ id: userObject.id }),
```

Comment:
```suggestion
			token: this.token.create({ userId: userObject.id }),
```
In #54 we use `userId`, let's have the same to avoid issues in future